Foundation/NSURLRequest.swift

```diff
@@ -364,11 +360,25 @@ public class NSMutableURLRequest : NSURLRequest {
         @result the HTTP request method of the receiver.
     */
     public override var HTTPMethod: String? {
-        get {
-            return _HTTPMethod
-        } set(newHTTPMethod) {
-            _HTTPMethod = newHTTPMethod
-        }
+        get { return super.HTTPMethod }
+        set { super.HTTPMethod = newValue }
+    }
+    
+    public override var cachePolicy: NSURLRequestCachePolicy {
+        get { return super.cachePolicy }
+        set { super.cachePolicy = newValue }
+    }
+    public override var timeoutInterval: NSTimeInterval {
+        get { return super.timeoutInterval }
+        set { super.timeoutInterval = newValue }
+    }
+    public override var networkServiceType: NSURLRequestNetworkServiceType {
+        get { return super.networkServiceType }
+        set { super.networkServiceType = newValue }
+    }
+    public override var allowsCellularAccess: Bool {
```

Comment:
can this ever be satisfied? It seems like it would require very specific hardware access